app/assets/stylesheets/views/_choose_sign_in.scss

```diff
@@ -0,0 +1,22 @@
+.verify-hint-box {
+  width: 100%;
+  border: 5px solid $govuk-border-colour;
+  padding: 15px;
```

Comment:
Whilst this is being torn down, so it's not too big a deal, spacing are generally done with [`govuk-spacing()` functions](https://design-system.service.gov.uk/styles/spacing/#spacing-on-custom-components).

For example, `15px` can be replaced with `govuk-spacing(3)` - which makes it easier to change in the future if the spacings are updated.